frontend/src/metabase/search/components/SearchResult/tests/SearchResult.unit.spec.tsx

```diff
@@ -58,4 +73,39 @@ describe("SearchResult", () => {
     ).not.toBeInTheDocument();
     expect(getIcon("folder")).toBeInTheDocument();
   });
+
+  it("should redirect to search result page when clicking item", async () => {
+    const { history } = setup({ result: TEST_RESULT_QUESTION });
+
+    userEvent.click(screen.getByText(TEST_RESULT_QUESTION.name));
+
+    const expectedPath = TEST_RESULT_QUESTION.getUrl();
+
+    expect(history?.getCurrentLocation().pathname).toEqual(expectedPath);
+  });
+
+  describe("indexed entities", () => {
+    it("renders x-ray button for indexed entity search result", () => {
+      setup({ result: TEST_RESULT_INDEXED_ENTITY });
+
+      expect(screen.getByTestId("search-result-item-icon")).toHaveAttribute(
+        "type",
+        "indexed-entity",
+      );
+
+      expect(screen.getByLabelText("bolt icon")).toBeInTheDocument();
```

Comment:
Nit: `getIcon` is our common helper for this, can be imported from `__support__/ui`

```suggestion
      expect(getIcon("bolt")).toBeInTheDocument();
```

Reply:
I'll make a quick cheeky PR to get this stuff in since the CI is passing, don't want to mess that up